Cargo.toml

```diff
@@ -27,6 +27,8 @@ sha1 = { version = "0.6", optional = true }
 md5 = { version = "0.3", optional = true }
 slog = { version = "2", optional = true }
 cfg-if = "0.1"
+uuid-macro-impl = { version = "0.1.0", path = "./crates/uuid-macro-impl" }
```

Comment:
make it `optional` and make `uuid!` accordingly behind a feature gate

Reply:
Thankyou for your addvice.